packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -784,7 +784,7 @@ static String interpretHeatmapOptions(Map<String, ?> data, HeatmapOptionsSink si
   }
 
   @VisibleForTesting
-  static List<LatLng> toPoints(Object o) {
+  static List<LatLng> pointsFromPigeon(Object o) {
```

Comment:
You've renamed the non-Pigeon version to say it's for Pigeon. The Pigeon version is the one that takes a `List<Messages.PlatformLatLng>`.

Reply:
Hm, I thought there were still unconverted codepaths that used `toPoints`, but now that you've changed the names it looks like there aren't; it's marked VisibleForTesting, and there are no references I see within the file. If `toPoints` is dead code now, it should be removed.

Reply:
I see a reference to it remaining at https://github.com/flutter/packages/blob/f8af5f5d71a2a6ce8938a9bc798ece2b9844d964/packages/google_maps_flutter/google_maps_flutter_android/android/src/test/java/io/flutter/plugins/googlemaps/ConvertTest.java#L98

Reply:
That's a unit test, not production code.